### PR TITLE
Addressing @Fabs comments

### PR DESCRIPTION
**Pull request by wavesoft**
This commit addresses all the comments from @Fabs from https://github.com/mesosphere/dcos-docs-site/pull/1587

## Jira Ticket
[DCOS-44080](https://jira.mesosphere.com/browse/DCOS-44080)

## Description of changes being made
Mostly lexical optimisation

## Checklist
- [X] Make sure you change all affected versions (e.g. 1.10, 1.11, 1.12, 1.13).
- [X] Test all commands and procedures where applicable.
- [X] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

### CODE REVIEW

File: pages/mesosphere/dcos/services/edge-lb/1.2/api-reference/edgelb/index.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The Edge-LB API also requires authorization via the following permissions:
 |-------|----------|
 | `/service/edgelb/` | `dcos:adminrouter:service:edge-lb` |
 
-All routes may also be reached by users with the `dcos:superuser` permission.
+Users with the dcos:superuser permissions may reach all routes.
```
Comment (orlandohohmeier) [inline pages/mesosphere/dcos/services/edge-lb/1.2/api-reference/edgelb/index.md]: I'd keep the formatting... s/dcos:superuser/`dcos:superuser`

File: pages/mesosphere/dcos/services/edge-lb/1.2/input-formats/index.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ enterprise: false
 ---
 
 
-Edge-LB accepts configuration files in either YAML or JSON format. Users are encouraged to use JSON and migrate their configuration from YAML format though, as YAML will be at some point deprecated.
+Edge-LB configuration files should preferably be written in JSON format. YAML is historically supported, but it will be deprecated in the future. We recommend users to migrate their YAML files to JSON
```
Comment (orlandohohmeier) [inline pages/mesosphere/dcos/services/edge-lb/1.2/input-formats/index.md]: Let's end the sentence with a period.

File: pages/mesosphere/dcos/services/edge-lb/1.2/release-notes/index.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ Edge-LB Service version 1.2.3 was released on November 27, 2018.
 
 * The steps provided in the DC/OS 1.10 web interface to uninstall Edge-LB are incorrect. In order to correctly uninstall Edge-LB for any given DC/OS version, please follow the steps in the [Edge-LB uninstall documentation](/mesosphere/dcos/services/edge-lb/1.2/uninstalling/).
 * Edge-LB running on a CentOS/RHEL 7.2 node where `/var/lib/mesos` is formatted with ext4 may have connection issues.
-* If a pool is configured with invalid constraints, that pool will not be properly created and will not respect pool deletion. It must be removed manually.
+* If a pool is configured with invalid constraints (e.g. some example if possible) it must be removed manually since a fail on proper creation leads to a failure in proper removal.
```
Comment (orlandohohmeier) [inline pages/mesosphere/dcos/services/edge-lb/1.2/release-notes/index.md]: I don't think we need an example here.

File: pages/mesosphere/dcos/services/edge-lb/1.2/release-notes/index.md [hunk 4]
```diff
@@ -82,4 +82,4 @@ Edge-LB Service version 1.2.2 was released on November 15, 2018.
...
```
Comment (orlandohohmeier) [inline pages/mesosphere/dcos/services/edge-lb/1.2/release-notes/index.md]: Again, I don't think we need an example here.

File: pages/mesosphere/dcos/services/edge-lb/1.3/api-reference/edgelb/index.md [hunk 5]
```diff
@@ -43,7 +43,7 @@ The Edge-LB API also requires authorization via the following permissions:
 |-------|----------|
 | `/service/edgelb/` | `dcos:adminrouter:service:edge-lb` |
 
-All routes may also be reached by users with the `dcos:superuser` permission.
+Users with the dcos:superuser permissions may reach all routes.
```
Comment (orlandohohmeier) [inline pages/mesosphere/dcos/services/edge-lb/1.3/api-reference/edgelb/index.md]: s/dcos:superuser/`dcos:superuser`

File: pages/mesosphere/dcos/services/edge-lb/1.3/release-notes/index.md [hunk 6]
```diff
@@ -39,7 +39,7 @@ Edge-LB Service version 1.3.1 was released April 29, 2019.
 
 - Edge-LB running on a CentOS/RHEL 7.2 node where `/var/lib/mesos` is formatted with `ext4` may have connection issues.
 
-- If a pool is configured with invalid constraints, that pool will not be properly created and will not respect pool deletion. It must be removed manually.
+- If a pool is configured with invalid constraints (e.g. some example if possible) it must be removed manually since a fail on proper creation leads to a failure in proper removal.
```
Comment (orlandohohmeier) [inline pages/mesosphere/dcos/services/edge-lb/1.3/release-notes/index.md]: No example needed here.

File: pages/mesosphere/dcos/services/edge-lb/1.3/release-notes/index.md [hunk 7]
```diff
@@ -77,4 +77,4 @@ Edge-LB Service version 1.3.0 was released February 6, 2019.
...
```
Comment (orlandohohmeier) [inline pages/mesosphere/dcos/services/edge-lb/1.3/release-notes/index.md]: Again, no example needed.

File: pages/mesosphere/dcos/services/edge-lb/1.4/api-reference/edgelb/index.md [hunk 8]
```diff
@@ -43,7 +43,7 @@ The Edge-LB API also requires authorization via the following permissions:
 |-------|----------|
 | `/service/edgelb/` | `dcos:adminrouter:service:edge-lb` |
 
-All routes may also be reached by users with the `dcos:superuser` permission.
+Users with the dcos:superuser permissions may reach all routes.
```
Comment (orlandohohmeier) [inline pages/mesosphere/dcos/services/edge-lb/1.4/api-reference/edgelb/index.md]: s/dcos:superuser/`dcos:superuser`

File: pages/mesosphere/dcos/services/edge-lb/1.4/release-notes/index.md [hunk 9]
```diff
@@ -50,7 +50,7 @@ Edge-LB Service version 1.3.1 was released April 29, 2019.
 
 - Edge-LB running on a CentOS/RHEL 7.2 node where `/var/lib/mesos` is formatted with `ext4` may have connection issues.
 
-- If a pool is configured with invalid constraints, that pool will not be properly created and will not respect pool deletion. It must be removed manually.
+- If a pool is configured with invalid constraints (e.g. some example if possible) it must be removed manually since a fail on proper creation leads to a failure in proper removal.
```
Comment (orlandohohmeier) [inline pages/mesosphere/dcos/services/edge-lb/1.4/release-notes/index.md]: No example needed.

File: pages/mesosphere/dcos/services/edge-lb/1.4/release-notes/index.md [hunk 10]
```diff
@@ -88,4 +88,4 @@ Edge-LB Service version 1.3.0 was released February 6, 2019.
...
```
Comment (orlandohohmeier) [inline pages/mesosphere/dcos/services/edge-lb/1.4/release-notes/index.md]: See above.

File: pages/mesosphere/dcos/services/edge-lb/1.2/api-reference/edgelb/index.md (hunk 1 above)
Comment (wavesoft) [reply pages/mesosphere/dcos/services/edge-lb/1.2/api-reference/edgelb/index.md]: whoops, indeed 👍

**Comment by wavesoft** [comment]
@ck4adventure poke? 👼

File: pages/mesosphere/dcos/services/edge-lb/1.2/api-reference/index.md [hunk 11]
```diff
@@ -8,4 +8,5 @@ excerpt: Reference for all API endpoints exposed by the Edge-LB package
 enterprise: false
 ---
 
-The API exposed to users consists of the Edge-LB API, which enables creation and manipulation of pools of load balancers and HAProxy API itself. Please choose a section on the left for details.
+The Edge-LB API is exposed to users allowing the creation and manipulation of pools of load balancers. Details of the different available APIs are available on the sections below.
```
Comment (ichernetsky) [inline pages/mesosphere/dcos/services/edge-lb/1.2/api-reference/index.md]: `on -> in`?

File: pages/mesosphere/dcos/services/edge-lb/1.2/pool-configuration/index.md [hunk 12]
```diff
@@ -12,20 +12,20 @@ This is a reference for all possible Edge-LB pool configuration values, as well
...
+A new top-level pool configuration field named `apiVersion` was introduced in Edge-LB v1.0.0. The two models are almost identical, with one important difference: `pool.haproxy.backends.servers` (in apiVersion `V1`) has been replaced with `pool.haproxy.backends.services`, with a more intuitive way to select services/backends for HAProxy.
 
-<p class="message--note"><strong>NOTE: </strong>Edge-LB 1.0 and later supports both the <tt>V1</tt> and <tt>V2</tt> API for backwards compatibility. Therefore clients that were written against Edge-LB versions prior to Edge-LB 1.0 should work without any modifications with Edge-LB 1.0 and later. New setups should use API <tt>V2</tt> as at some point <tt>V1</tt> is going to be deprecated and then removed.</p>
+<p class="message--note"><strong>NOTE: </strong>Edge-LB 1.0 and later supports both the <tt>V1</tt> and <tt>V2</tt> API for backward compatibility. Therefore clients that were written against Edge-LB versions prior to Edge-LB 1.0 should work without any modifications with Edge-LB 1.0 and later. New setups should use API <tt>V2</tt> as at some point <tt>V1</tt> is going to be deprecated and then removed.</p>
```
Comment (ichernetsky) [inline pages/mesosphere/dcos/services/edge-lb/1.2/pool-configuration/index.md]: nit: I think both are correct.

File: pages/mesosphere/dcos/services/edge-lb/1.2/release-notes/index.md [hunk 13]
```diff
@@ -34,7 +34,7 @@ Edge-LB Service version 1.2.3 was released on November 27, 2018.
 
 * The steps provided in the DC/OS 1.10 web interface to uninstall Edge-LB are incorrect. In order to correctly uninstall Edge-LB for any given DC/OS version, please follow the steps in the [Edge-LB uninstall documentation](/mesosphere/dcos/services/edge-lb/1.2/uninstalling/).
 * Edge-LB running on a CentOS/RHEL 7.2 node where `/var/lib/mesos` is formatted with ext4 may have connection issues.
-* If a pool is configured with invalid constraints, that pool will not be properly created and will not respect pool deletion. It must be removed manually.
+* If a pool is configured with invalid constraints it must be removed manually since a fail on proper creation leads to a failure in proper removal.
```
Comment (ichernetsky) [inline pages/mesosphere/dcos/services/edge-lb/1.2/release-notes/index.md]: failure?

File: pages/mesosphere/dcos/services/edge-lb/1.2/release-notes/index.md [hunk 14]
```diff
@@ -82,4 +82,4 @@ Edge-LB Service version 1.2.2 was released on November 15, 2018.
...
```
Comment (ichernetsky) [inline pages/mesosphere/dcos/services/edge-lb/1.2/release-notes/index.md]: ditto

File: pages/mesosphere/dcos/services/edge-lb/1.3/api-reference/index.md [hunk 15]
```diff
@@ -8,4 +8,4 @@ excerpt: Reference for all API endpoints exposed by the Edge-LB package
 enterprise: false
 ---
 
-The API exposed to users consists of the Edge-LB API, which enables creation and manipulation of pools of load balancers and HAProxy API itself. Please choose a section on the left for details.
+The Edge-LB API is exposed to users allowing the creation and manipulation of pools of load balancers. Details of the different available APIs are available on the sections below.
```
Comment (ichernetsky) [inline pages/mesosphere/dcos/services/edge-lb/1.3/api-reference/index.md]: ditto

File: pages/mesosphere/dcos/services/edge-lb/1.4/api-reference/index.md [hunk 16]
```diff
@@ -8,4 +8,4 @@ excerpt: Reference for all API endpoints exposed by the Edge-LB package
 enterprise: false
 ---
 
-The API exposed to users consists of the Edge-LB API, which enables creation and manipulation of pools of load balancers and HAProxy API itself. Please choose a section on the left for details.
+The Edge-LB API is exposed to users allowing the creation and manipulation of pools of load balancers. Details of the different available APIs are available on the sections below.
```
Comment (ichernetsky) [inline pages/mesosphere/dcos/services/edge-lb/1.4/api-reference/index.md]: ditto

File: pages/mesosphere/dcos/services/edge-lb/1.4/pool-configuration/index.md [hunk 17]
```diff
@@ -11,20 +11,20 @@ This section provides reference information for all Edge-LB pool configuration s
 
 # Configuration reference
 
-The API reference, in swagger format, containing all possible options and short descriptions of each endpoint can be found by running the following command:
+The API reference, in swaggger format, can be found by running the command bellow. It contains all possible options and a short description for each endpoint
```
Comment (ichernetsky) [inline pages/mesosphere/dcos/services/edge-lb/1.4/pool-configuration/index.md]: `below`

Comment (ichernetsky) [inline pages/mesosphere/dcos/services/edge-lb/1.4/pool-configuration/index.md]: Should the be a point or color at the end?

**Comment by cmastr** [comment]
@orlandohohmeier and @wavesoft do we want a new PR for the universe changes or can you make reqd changes to this PR?

**Comment by wavesoft** [comment]
Going to change them now...

**Comment by wavesoft** [comment]
Hey @cmastr ! I thought we are now calling it "catalog" instead of "universe" but I think I have misunderstood your suggestion.

Note that edge-LB is *not* published in the DCOS catalog, so I am not sure what to fix here?

**Comment by SarahStegall** [comment]
> Hey @cmastr ! I thought we are now calling it "catalog" instead of "universe" but I think I have misunderstood your suggestion.
> 
> Note that edge-LB is _not_ published in the DCOS catalog, so I am not sure what to fix here?

Hi Ioannis. The Edge-LB documentation contains references to the DC/OS Catalog, but those references sometimes use the terminology "Universe", which is out of date. For example, on this page:

/edge-lb/1.5/getting-started/installing/

you will see that there are references to "Universe". These need to be changed to "Catalog", otherwise users can get confused, thinking that Universe and Catalog are different things.

I believe Casie is asking if you are willing to go through the Edge-LB documentation and replace the references to "Universe" to "Catalog". (Obviously this would not apply to any URLs or code that still say `universe`.)

**Comment by wavesoft** [comment]
Ok, I see .. I thought that we call the upstream universe the "Service Catalog", but I forgot we also call the UI component "Catalog".

Even though it feels a bit weird to me to mix the naming, I have just pushed the changes requested. Let me know if that's what you meant 😄

**Comment by wavesoft** [comment]
@SarahStegall , @cmastr let me know if there is anything else you need.

**Pull request merged**